deploy/generate-deployment.sh

```diff
@@ -69,7 +69,7 @@ done
 
 if $USE_DEFAULT_ENV; then
   echo "Using defaults for environment variables"
-  export NAMESPACE=devworkspace-controller
+  export NAMESPACE=openshift-operators
```

Comment:
IMO this is the only change that should be made; the namespace in the Makefile will be ignored when the `deployment/deploy` templates are generated anyways.

Reply:
I would argue on that.
I mean I would be happy if we can reuse Makefile to work with deployment run with chectl. And Makefile reuse NAMESPACE from there to restart deployments at least.

But as we discussed on standup, probably we should stick to `devworkspace-controller` namespace as only available, that would work for K8s and Openshift and easier to clean up.

Reply:
Using the Makefile in `chectl` means having `chectl` depend on `make`, which is a strange choice. If we need a development version of DWO to run with Che, it should be deployed manually from our repo.

Reply:
I meant you deploy it with chectl, but then you need to update/debug DevWorkpaceOperator with makefile.
if we unify values, it would be possible to deploy/update in any case you like.